app/resources/ama_resource.rb

```diff
@@ -0,0 +1,10 @@
+class AMAResource < BaseResource
+  attributes :description, :start_date, :end_date,
+    :ama_subscribers_count
+
+  has_one :author
+  has_one :original_post
+  has_many :ama_subscribers
+
+  filters :end_date, :start_date, :ama_subscribers_count
```

Comment:
I don't think JR provides this by default. We've implemented it manually in `MediaResource`. for AMAs tho we don't need it to be that precise, so I think simple past and future filters would suffice.